.github/workflows/build-docker-image.yml

```diff
@@ -38,9 +38,72 @@ jobs:
       uses: docker/build-push-action@v2
       with:
         context: .
-        file: ./Dockerfile
+        file: ./linux/Dockerfile
+        # In case linux/s390x fails with tls issues:
+        # see https://github.com/testcontainers/moby-ryuk/pull/26
+        # and https://github.com/testcontainers/moby-ryuk/pull/40
         platforms: linux/amd64,linux/arm/v7,linux/arm64,linux/ppc64le,linux/s390x
         # Only push if we are publishing a release
         push: ${{ github.event_name == 'release' && github.event.action == 'published' }}
         # Use a 'temp' tag, that won't be pushed, for non-release builds
         tags: testcontainers/ryuk:${{ github.event.release.tag_name || 'temp' }}
+
+  build-windows:
+    runs-on: windows-2022
+    steps:
+    - uses: actions/checkout@v3
+    - name: Login to Docker Hub
+      # Only if we need to push an image
+      if: ${{ github.event_name == 'release' && github.event.action == 'published' }}
+      uses: docker/login-action@v2
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+    # Debugging information
+    - name: Docker info
+      run: docker info
+
+    - name: Build image
+      # Use a 'temp' tag, that won't be pushed, for non-release builds
+      run: |
```

Comment:
`docker/*` actions instead of the CLI?

Reply:
I think the docker/build-and-push-action wouldn't work for Windows, see https://github.com/docker/build-push-action/issues/18.

Reply:
In fact, I tried without actually planning: see https://github.com/testcontainers/moby-ryuk/pull/40/commits/54947b0125d55584bc97bd81d3dd6f42f776112e#diff-5b40193e09025fd041de92d38c976d0b469e3ee85bef45a71d05d8b5078b1ee8L66, which failed with an error indicating that buildx wasn't compatible.

Reply:
Ok, wasn't aware of that. I assumed similar capabilities as the Azure Develops tasks have.

Reply:
I need to check whether https://lippertmarkus.com/2021/11/30/win-multiarch-img-lin/ helps.